scss/templates/user.scss

```diff
@@ -198,3 +148,188 @@
     }
   }
 }
+
+.stats-section {
+  .stats-title {
+    display: none;
+  }
+
+  ul {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  li,
+  .linked-stat {
+    margin: 0;
+    padding: 0;
+    margin: 0 1.5rem 1.5rem 0;
+
+    > a {
+      padding: 0 !important;
+    }
+  }
+
+  .user-stat {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    width: 10rem;
+    height: 7.5rem;
+    background-color: $blue;
+    border-radius: 0.25rem;
+
+    .number,
+    .label,
+    .value {
+      color: $white;
+      font-style: normal;
+      font-weight: normal;
+    }
+
+    .number,
+    .value {
+      font-size: 2.25rem;
+      font-family: "Roboto", $font-family-sans-serif;
+    }
+
+    .label {
+      font-size: 0.75rem;
+      font-weight: bold;
+      text-transform: uppercase;
+      margin-top: 0.75rem;
+
+      .d-icon-heart {
+        display: none;
+      }
+    }
+  }
+
+  @include media-breakpoint-down(sm) {
+    ul {
+      display: grid;
+      grid-template-columns: repeat(2, 1fr);
+      grid-gap: 0.75rem;
+    }
+
+    li,
+    .linked-stat {
+      margin: 0;
+    }
+
+    .user-stat {
+      width: 100%;
+    }
+  }
+}
+
+.most-liked-section,
+.top-replies-section {
+  display: grid;
+  grid-gap: 2.4375rem;
+  grid-template-columns: 1fr;
+
+  .stats-title {
+    text-transform: none;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 1.125rem;
+    line-height: 1.375rem;
+    color: $gray-100;
+  }
+
+  .top-sub-section,
+  .summary-user-list {
+    width: 100%;
+    margin-bottom: 0;
+
+    > div > ul > li,
```

Comment:
I will rather add more specificity like:

```
.top-sub-section,
.summary-user-list {
    &.summary-user-list li,
    &.top-sub-section li {
        //...
    }   
}
```

Still a bit odd but we are not that tight to DOM structure. _(same for other rules with similar approach)_